quickstart.md

```diff
@@ -270,12 +270,284 @@ In case the user wants to run the workload on a TDX capable hardware, using QEMU
+
+  ```
+  sudo apt install docker-compose
+  ```
+
+Clone the repository for specified tag:
+```
+simple_kbs_tag="0.1.1"
+git clone https://github.com/confidential-containers/simple-kbs.git
+(cd simple-kbs && git checkout -b "branch_${simple_kbs_tag}" "${simple_kbs_tag}")
+```
+
+Run the service with `docker-compose`:
+```
+(cd simple-kbs && sudo docker-compose up -d)
+```
+
+#### Calculate the SEV Guest Measurement and Store the simple-kbs Policy
+
+This step describes the process to calculate the SEV guest firmware measurement and create a key release policy in the `simple-kbs` requiring the kata shim to provide a matching measurement.
+
+`pip` is required to install the `sev-snp-measure` utility. `mysql-client` is required to insert the key release policy into the `simple-kbs` database. `jq` is required to json parse responses on the command line.
+
+* Debian Ubuntu:
+
+  ```
+  sudo apt install -y python3-pip mysql-client jq
+  ```
+
+[sev-snp-measure](https://github.com/IBM/sev-snp-measure) is a utility used to calculate the SEV guest firmware measurement with provided ovmf, initrd, kernel and kernel append input parameters. Install it using the following command:
+
+```
+sudo pip install sev-snp-measure
+```
+
+The path to the guest binaries required for measurement is specified in the kata configuration. Set them:
+
+```
+ovmf_path="/opt/confidential-containers/share/ovmf/OVMF.fd"
+kernel_path="/opt/confidential-containers/share/kata-containers/vmlinuz-sev.container"
+initrd_path="/opt/confidential-containers/share/kata-containers/kata-containers-initrd.img"
+```
+
+The kernel append line parameters are included in the SEV guest firmware measurement. Provided here is a common value for these parameters:
+
+```
+append="tsc=reliable no_timer_check rcupdate.rcu_expedited=1 \
+i8042.direct=1 i8042.dumbkbd=1 i8042.nopnp=1 i8042.noaux=1 noreplace-smp reboot=k \
+cryptomgr.notests net.ifnames=0 pci=lastbus=0 console=hvc0 console=hvc1 quiet panic=1 \
+nr_cpus=1 scsi_mod.scan=none agent.config_file=/etc/agent-config.toml"
+```
+
+Use the `sev-snp-measure` utility to calculate the SEV guest firmware measurement using the binary variables previously set:
+
+```
+measurement=$(sev-snp-measure --mode=sev --output-format=base64 \
+  --ovmf="${ovmf_path}" \
+  --kernel="${kernel_path}" \
+  --initrd="${initrd_path}" \
+  --append="${append}" \
+)
+```
+
+The `simple-kbs` uses default settings and credentials for the MySQL database. These settings can be changed by the `simple-kbs` administrator and saved into a credential file. For the purposes of this quick start, set them in the environment for use with the MySQL client command line:
+
+```
+KBS_DB_USER="kbsuser"
+KBS_DB_PW="kbspassword"
+KBS_DB="simple_kbs"
+KBS_DB_TYPE="mysql"
+```
+
+Retrieve the host address of the MySQL database container:
+
+```
+KBS_DB_HOST=$(sudo docker network inspect simple-kbs_default \
+  | jq -r '.[].Containers[] | select(.Name | test("simple-kbs_db.*")).IPv4Address' \
```

Comment:
That `jq` does not seem to work on RHEL. I do have an address, but `jq` gives me nothing:

```
# sudo docker network inspect simple-kbs_default   G IPv4Address
            "IPv4Address": "172.18.0.2/16",
            "IPv4Address": "172.18.0.3/16",
# sudo docker network inspect simple-kbs_default   | jq -r '.[].Containers[] | select(.Name | test("simple-kbs_db.*")).IPv4Address'
# 
```

Reply:
I have this from the `Containers` step:
```
 sudo docker network inspect simple-kbs_default   | jq -r '.[].Containers[] '         
{
  "Name": "simple-kbs-db-1",
  "EndpointID": "f0492a5216f4a84f080cc1330defea1e9a711705b612576c3ca622b519bfceb5",
  "MacAddress": "02:42:ac:12:00:02",
  "IPv4Address": "172.18.0.2/16",
  "IPv6Address": ""
}
{
  "Name": "simple-kbs-server-1",
  "EndpointID": "5dbca9f63ba16cff7314675c60b61ae212ee2eddac6d7906faac488ac047ba86",
  "MacAddress": "02:42:ac:12:00:03",
  "IPv4Address": "172.18.0.3/16",
  "IPv6Address": ""
}
```

Reply:
```suggestion
  | jq -r '.[].Containers[] | select(.Name | test("simple-kbs-db.*")).IPv4Address' \
 ```

Reply:
added a regex here to handle both cases. We've seen it change between a `_` and a `-`.